src/universal/modules/welcome/ducks/__tests__/welcomeDuck-tests.js

```diff
@@ -68,3 +74,11 @@ test('previousPage() is never less than one', t => {
     page: 1
   });
 });
+
+test('updateCompleted() updates state', t => {
```

Comment:
@ackernaut I just noticed I didn't write a test for UPDATED_COMPLETED. Perhaps you want to try your hand at that here?